### ISSUE TITLE
how to specify whether or not to push new changes

### ISSUE THREAD

**Issue by silverag-corgi** [python-semantic-release/python-semantic-release#574]
Hello!

Thanks for wonderful tool!
I want to ask about tool specification.
- tool version tag: `v7.33.2`

When running command `semantic-release publish`,
is there an option like `push_changelog_and_tag` that specifies whether or not to push new changes?
I want to check created changelog and version tag, and then optionally push.
Have you not considered this case?

Also, I know that setting `false` to `commit_version_number` and `tag_commit` will update only changelog file and version of `pyproject.toml`.

And this issue is similar to #381, which is unresolved.
In other words, are you going to implement it but not yet,
or for example because of the ideology of this tool, are you not going to implement it in the first place?

Thanks for reading!

**Comment by bernardcooke53** [comment]
Hi @silverag-corgi :wave:

This isn't yet an option but it will be available in a v8 release, you will be able to do something like:

```shell
semantic-release version --no-push
```

Which should do what you're looking for.

**Comment by silverag-corgi** [comment]
Hi!
I see, it is just in the middle of developing!
That's nice:smile:
I think this issue is closed.
Thanks.
